application/src/main/kotlin/no/nav/poao_tilgang/application/utils/JsonUtils.kt

```diff
@@ -10,6 +10,8 @@ import com.fasterxml.jackson.module.kotlin.treeToValue
 
 object JsonUtils {
 
+	//TODO burde denne flyttes til api-core-mapper? da har vi bare en instans av objectmapper
+	//eksisiterer også en instangs av objectmapper i api-core-mapper
 	val objectMapper: ObjectMapper = ObjectMapper()
```

Comment:
Det finnes 3 stk nå av den - gjør det ikke? også en i clientpakka, så om vi kan flytte denne eller den andre tenker jeg nok det kan være en fordel ja :)

Reply:
Da er det nok greit som det er :)

Reply:
Vær litt obs på at det er en viss risiko for deadlock-situasjoner dersom du bruker samme objectmapper overalt, selv om den er thread-safe.
https://stackoverflow.com/questions/3907929/should-i-declare-jacksons-objectmapper-as-a-static-field